tests/JsonApiSerializerTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace Spatie\Fractal\Test;
+
+use League\Fractal\Serializer\JsonApiSerializer;
+
+class JsonApiSerializerTest extends TestCase
+{
+    public function setUp($defaultSerializer = '')
+    {
+        parent::setUp();
+
+        app('config')->set('laravel-fractal.default_serializer', JsonApiSerializer::class);
+    }
+
+    /** @test */
+    public function it_cannot_has_links_when_json_api_base_url_is_null_on_instantiate()
```

Comment:
The language used in the function names of the new tests is a bit off.

This could be named `it_will_not_generate_links_when_the_base_url_is_null` or something better.